packages/react-day-picker/src/hooks/useDayRender/useDayRender.tsx

```diff
@@ -99,8 +101,16 @@ export function useDayRender(
     'aria-label': ariaLabel
   };
 
+  const isOutsideAndDisplayed =
+    activeModifiers.outside && navigation.isDateDisplayed(day);
+
+  // If a day is both outside and displayed, the equivalent "inside" day must also be present on one
+  // of the displayed calendars, and we do not want to have multiple days with tabIndex="0" because
+  // it can break tabbed navigation (https://github.com/gpbl/react-day-picker/issues/1567)
   const isFocusTarget = Boolean(
-    focusContext.focusTarget && isSameDay(focusContext.focusTarget, day)
+    focusContext.focusTarget &&
+      isSameDay(focusContext.focusTarget, day) &&
+      !isOutsideAndDisplayed
```

Comment:
I think we don't need to check if the date is displayed or not. I'd just check the `activeModifiers.outside`.

```suggestion
      !activeModifiers.outside
```